semanticdb/metai/src/main/scala/scala/meta/internal/metai/Main.scala

```diff
@@ -25,19 +25,18 @@ final class Main(settings: Settings, reporter: Reporter) {
     val status = mutable.Map[AbsolutePath, Boolean]()
     val job = Job(settings.classpath.entries, if (settings.verbose) reporter.err else devnull)
     job.foreach { entry =>
-      Classpath(entry).foreach { entry =>
-        val success = {
-          try {
-            processEntry(entry)
-          } catch {
-            case e: Throwable =>
-              reporter.err.println(s"Error indexing ${entry.pathOnDisk}:")
-              e.printStackTrace(reporter.err)
-              false
-          }
+      var success = true
+      try {
+        Classpath(entry).foreach { entry =>
```

Comment:
Note that this changes the behaviour from error accumulation to failing on first error.